src/AppFrame.tsx

```diff
@@ -47,18 +112,13 @@ export function AppFrame() {
                 label: t("home"),
                 icon: HomeMajor,
               },
-              {
-                label: languageToggle(),
-                onClick: () => {
-                  if (i18n.language?.substring(0, 2) === "fr") {
-                    i18next.changeLanguage("en-CA");
-                  } else {
-                    i18next.changeLanguage("fr-CA");
-                  }
-                },
-              },
             ]}
           />
+          <Navigation.Section // Todo translating the word "Language" and displaying icon doesn't work for some reason
```

Comment:
`title={t("language")}` doesn't work?

Reply:
I'm not sure why this doesn't work, seems to be a Shopify Polaris issue. Can you figure out a workaround? Also I was hoping to put a language icon here but that doesn't work either.

Reply:
Localizing the text works for me:

![image](https://user-images.githubusercontent.com/2429373/119261258-98f29f00-bba4-11eb-9b1c-00eb836bb224.png)

Reply:
Fixed. It won't let me put an icon in though which I wanted to do

Reply:
How were you trying to add an icon?

Reply:
I tried icon={LanguageMinor} and the icon doesn't show up - was hoping we could put an icon beside Language so it's obvious even if you don't speak English what it means

Reply:
translating the string "Language" does work though